src/config/groups/keys.cpp

```diff
@@ -1286,7 +1286,7 @@ std::pair<std::string, ustring> Keys::decrypt_message(ustring_view ciphertext) c
     //
     // Removing any null padding bytes from the end
     //
-    if (auto pos = plain.find_last_not_of('\0'); pos != std::string::npos)
+    if (auto pos = plain.find_last_not_of((unsigned char) 0); pos != std::string::npos)
```

Comment:
That should be able to be just a plain `0` (i.e. I don't think the explicit `(unsigned char)` cast is needed).  If compilers are warning about that, though, then `unsigned char{0}` is a slightly cleaner way to write it.

Reply:
Just tried making is just `0`, but I get 
```
D:\a\libsession-util-nodejs\libsession-util-nodejs\libsession-util\src\config\groups\keys.cpp(1289,26): error C2668: 'std::basic_string<unsigned char,std::char_traits<unsigned char>,std::allocator<unsigned char>>::find_last_not_of': ambiguous call to overloaded function [D:\a\libsession-util-nodejs\libsession-util-nodejs\build\libsession-util\src\config.vcxproj]

```
for all builds
https://github.com/oxen-io/libsession-util-nodejs/actions/runs/9591309454/job/26447994842

Reply:
I had to make it 
`(unsigned char){0}`
as I'd get `type  name is not allowed`
without the brackets

Reply:
actually, msvc doesn't like this, so reverting to the good ol' `unsigned char(0)`
https://github.com/oxen-io/libsession-util-nodejs/actions/runs/9591425021/job/26448313193